truffleruby/src/main/ruby/core/rbconfig.rb

```diff
@@ -148,6 +144,11 @@ def RbConfig.expand(val, config = CONFIG)
     val.replace(newval) unless newval == val
     val
   end
+
+  CONFIG.each_value do |val|
+    RbConfig::expand(val)
+  end
```

Comment:
@bjfish Is there any advantage to have these variables available in the Makefile?
Otherwise we could just do something like `MAKEFILE_CONFIG = CONFIG.dup`.